karma.conf.js

```diff
@@ -21,16 +21,14 @@ webpackConfig.resolve.alias['stripes-config'] = path.resolve(__dirname, 'tests/s
 // `stripes-config` module does not get overriden.
 // see `@folio/stripes-config`
 // see `@folio/stripes-core/webpack.config.base.js`
-webpackConfig.module.rules = webpackConfig.module.rules.filter(rule => {
-  return !rule.use || !rule.use.some(use => use.loader === '@folio/stripes-config');
-});
+webpackConfig.module.rules = webpackConfig.module.rules.filter(rule => !rule.use || !rule.use.some(use => use.loader === '@folio/stripes-config'));
```

Comment:
Whoa, this is kinda hard to read imo. Can we make this more readable but still respect eslint?